src/types.ts

```diff
@@ -90,6 +90,7 @@ export type SubCommandsDef = Record<string, Resolvable<CommandDef<any>>>;
 export type CommandDef<T extends ArgsDef = ArgsDef> = {
   meta?: Resolvable<CommandMeta>;
   args?: Resolvable<T>;
+  alias?: Resolvable<string | string[]>;
```

Comment:
minor nit, maybe this should actually be in a `SubCommandDef`

you can't alias the root command since it is the 'argless' execution afaict

so maybe `SubCommandsDef` should actually be `Record<string, Resolvable<SubCommandDef<any>>>`, where `SubCommandDef` is just `CommandDef<T> & {alias?: Resolvable<string | string[]>}`

what do you think?